bpf/bpf_host.c

```diff
@@ -1019,7 +1019,28 @@ int cil_from_netdev(struct __ctx_buff *ctx)
 
 	if (flags & XFER_PKT_ENCAP) {
 		edt_set_aggregate(ctx, 0);
-
+#if defined(ENABLE_DSR) && DSR_ENCAP_MODE == DSR_ENCAP_GENEVE
+		{
+			struct geneve_opt4 gopt;
+			__u16 port = (__u16)ctx_get_xfer(ctx, XFER_ENCAP_PORT);
+			__u32 addr = ctx_get_xfer(ctx, XFER_ENCAP_ADDR);
```

Comment:
These are `__be16` / `__be32`, right? Just want to make sure that we have the endianess consistent everywhere.

Reply:
Yes, I will fix it